config/rbac/kustomization.yaml

```diff
@@ -3,10 +3,11 @@ resources:
 - role_binding.yaml
 - leader_election_role.yaml
 - leader_election_role_binding.yaml
+- serviceaccount.yaml
 # Comment the following 4 lines if you want to disable
 # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
 # which protects your /metrics endpoint.
-- auth_proxy_service.yaml
-- auth_proxy_role.yaml
-- auth_proxy_role_binding.yaml
-- auth_proxy_client_clusterrole.yaml
+#- auth_proxy_service.yaml
+#- auth_proxy_role.yaml
+#- auth_proxy_role_binding.yaml
+#- auth_proxy_client_clusterrole.yaml
```

Comment:
Does this mean the /metrics endpoint is now unprotected?

Reply:
Yes and no, as it also removes the service that pointed to the metrics endpoint so it's not reachable at all. Consuming this in a prometheus rule would probably be good to gate behind a value in an uncoming helm chart (there's another PR for that), or if someone was to look at the kustomization themselves they would be able to uncomment it and re-add it.